code/game/atoms.dm

```diff
@@ -752,3 +760,52 @@ var/list/blood_splatter_icons = list()
 
 /atom/Exited(atom/movable/AM, atom/newLoc)
 	SEND_SIGNAL(src, COMSIG_ATOM_EXITED, AM, newLoc)
+
+/*
+	Adds an instance of colour_type to the atom's atom_colours list
+*/
+/atom/proc/add_atom_colour(coloration, colour_priority)

```

Comment:
Another back-end port from /tg/. Ideally we want to move all of our color changes here as well, but that's for a future PR.